### ISSUE TITLE
Broken link on Heroku?

### ISSUE THREAD

**Issue by msmichellegar** [liberty-x/lxissue#30]
After clicking "log in with GitHub", I got the following :(

![screen shot 2015-10-31 at 12 00 09 pm](https://cloud.githubusercontent.com/assets/10683087/10863481/1759ed1e-7fc7-11e5-8087-7809490c1bdb.png)

Perhaps a link leftover from running it on localhost?

**Comment by hdrdavies** [comment]
@msmichellegar  that's correct, we switched it to localhost so we could test it and improve code, so now we should really switch it back.
